src/sentry/api/serializers/models/release.py

```diff
@@ -343,6 +345,9 @@ def get_attrs(self, item_list, user, **kwargs):
             p.update(release_metadata_attrs[item])
             p.update(deploy_metadata_attrs[item])
 
+            if with_file_count:
+                p["file_count"] = ReleaseFile.objects.filter(release=item).count()
```

Comment:
Can this lead to the N+1 problem / should we do it in a more efficient way?

Reply:
Could you do a query that generates a count for each release? Something like

```python
totals = Release.objects
  .filter(id__in=[r.id for r in item_list])
  .annotate(file_count=Count('releasefile'))
  .values_list('id', 'file_count')
```

You could map that result into a dictionary of releaseid : file_count.

Reply:
Thanks, I did it that way